src/test/java/mate/academy/dao/impl/RoleDaoImplTest.java

```diff
@@ -0,0 +1,39 @@
+package mate.academy.dao.impl;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.util.NoSuchElementException;
+import mate.academy.dao.RoleDao;
+import mate.academy.model.Role;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RoleDaoImplTest extends AbstractTest {
```

Comment:
You have 3 methods in RoleDao but here only one is tested. Add more tests.
Same for UserDaoImplTest.